config/environments/test.rb

```diff
@@ -12,7 +12,7 @@
   # Settings specified here will take precedence over those in config/application.rb.
 
   # https://guides.rubyonrails.org/configuring.html#config-cache-classes
-  config.cache_classes = true
+  config.cache_classes = false
```

Comment:
turning this off because `with_modal` gem-related code will fail. I believe they will patch it.

This slow our the CI suite a little bit.